client/src/main/java/io/spine/client/package-info.java

```diff
@@ -19,9 +19,34 @@
  */
 
 /**
- * This package provides utility classes for client-side code.
+ * This package provides classes and interfaces for performing client requests like
+ * posting commands, running queries, or creating subscriptions.
+ *
+ * <p>The term <em>actor</em> used in this API means the user on behalf of whom requests
+ * are created and executed. The package provides two levels of API.
+ *
+ * <dl>
+ *   <dt><strong>High-level {@link io.spine.client.Client} API</strong></dt>
+ *   <dd>is meant for client-side Java applications that would communicate with backend
+ *   services via a gRPC connection. An instance of the {@link io.spine.client.Client} class
+ *   establishes this connection and serves as a fluent API gateway for composing and posting
+ *   requests.
+ *   </dd>
+ *
+ *   <dt><strong>Low-level {@link io.spine.client.ActorRequestFactory} API</strong></dt>
```

Comment:
Do we need `strong` if we've already goit `dt`?

Reply:
We do. This way the tagged text comes in bold, which makes it look like a header. Otherwise, it's the same font as the rest of the text. The `dt/dd` markup is just semantics, and in Javadoc we're limited with how we apply styles.

I've tried headers, and they are way too big for this docs. Have a look like Guava does package docs.